sdk/resource/internal/schema/generate/schema/fetch.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
Can you please add some package documentation that would describe what this tool is doing and why it was created.

Also am I correct the usage of it is not currently implemented? Shouldn't it be invoked as part of `make semconv-generate`?

Reply:
I'll update RELEASING.md to include it as a manual step for now.